watcher.js

```diff
@@ -0,0 +1,27 @@
+
+
+var wsReloader;
+var LAST_SUCCESS_BUILD_STAMP = (localStorage.getItem('LAST_SUCCESS_BUILD_STAMP') || 0)
+var WS_PORT = 9999; // configurable
+
+function setUpWebScoket() {
+    if (wsReloader == null || wsReloader.readyState !== 1) {
+        try {
+            wsReloader = new WebSocket(`ws://localhost:${WS_PORT}`)
+            wsReloader.onmessage = (msg) => {
+                var newData = JSON.parse(msg.data).LAST_SUCCESS_BUILD_STAMP
+                if (newData > LAST_SUCCESS_BUILD_STAMP) {
+                    LAST_SUCCESS_BUILD_STAMP = newData
+                    localStorage.setItem('LAST_SUCCESS_BUILD_STAMP', LAST_SUCCESS_BUILD_STAMP)
+                    location.reload(true)
+                }
+
+            }
+        } catch (exn) {
+            console.error("web socket failed connect")
+        }
+    }
+};
+
+setUpWebScoket();
```

Comment:
@jchavarri I doubt this is your mistake, but spelling here is questionable ;-)

Reply:
Yeah, this comes from the `bsb` themes 😅 fixed it in the [last commit](https://github.com/ahrefs/atdgen-workshop-starter/pull/1/commits/889fbd08082d72c25306a9c02c170c8b2dd5277f).